main.go

```diff
@@ -0,0 +1,193 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"runtime"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/fileutil"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-tools/go-steputils/stepconf"
+	semver "github.com/hashicorp/go-version"
+)
+
+// Config model
+type Config struct {
+	Workdir    string `env:"workdir"`
+	Command    string `env:"command,required"`
+	NpmVersion string `env:"npm_version"`
+}
+
+func getNpmVersionFromPackageJSON() (string, error) {
```

Comment:
this function should consider using the Workdir, may the package.json is not in the current directory